source/VirtualScroll/VirtualScroll.js

```diff
@@ -141,6 +142,7 @@ export default class VirtualScroll extends Component {
           stopIndex: rowStopIndex
         })}
         overscanRowCount={overscanRowCount}
+        removeHeightContainer={removeHeightContainer}
```

Comment:
Could you add this (or rather, the renamed `autoHeight`) pass-thru property to `FlexTable` as well?